.github/workflows/deprecate.yml

```
@@ -0,0 +1,27 @@
+name: "Deprecate package"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'type version: x.y.z (without "v")'
+        required: false
+      reason:
+        description: "deprecation message (leave empty to un-deprecate)"
+        required: false
```

Возможность раздепрекейтить пакет выглядит сомнительной, за все это время не могу припомнить хоть одного кейса, где это было бы надо.

```suggestion
        description: "reason for deprecation"
        required: true
```